modules/nf-core/fastqc/meta.yml

```
@@ -1,67 +1,67 @@
-name: fastqc
-description: Run FastQC on sequenced reads
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "fastqc"
+## TODO nf-core: Add a description of the module and list keywords
+description: write your description here
 keywords:
-  - quality control
-  - qc
-  - adapters
-  - fastq
+  - sort
+  - example
+  - genomics
 tools:
-  - fastqc:
-      description: |
-        FastQC gives general quality metrics about your reads.
-        It provides information about the quality score distribution
-        across your reads, the per base sequence content (%A/C/G/T).
-
-        You get information about adapter contamination and other
-        overrepresented sequences.
-      homepage: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/
-      documentation: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/
-      licence: ["GPL-2.0-only"]
+  - "fastqc":
+      ## TODO nf-core: Add a description and other details for the software below
+      description: "A quality control tool for high throughput sequence data."
+      homepage: "None"
+      documentation: "None"
+      tool_dev_url: "None"
+      doi: ""
+      licence: ["GPL >=3"]
       identifier: biotools:fastqc
+
+## TODO nf-core: Add a description of all of the variables used as input
 input:
   - - meta:
         type: map
         description: |
           Groovy Map containing sample information
-          e.g. [ id:'test', single_end:false ]
-    - reads:
+          e.g. `[ id:'sample1', single_end:false ]`
```

Can we remove this from the template? It isn't often required and gets left and clutters things up.
```suggestion
          e.g. `[ id:'sample1]`
```